necessist/tests/necessist_db_absent/cli_dir_input-mixed.toml

```
@@ -0,0 +1,11 @@
+args = ["--no-sqlite", "--dump-candidates", "--framework=rust", "--root=fixtures/cli_dir_input", "fixtures/cli_dir_input/src/target_dir", "fixtures/cli_dir_input/src/adjacent_dir/src/lib.rs"]
```

```suggestion
args = ["--no-sqlite", "--dump-candidates", "--root=fixtures/cli_dir_input", "fixtures/cli_dir_input/src/target_dir", "fixtures/cli_dir_input/src/adjacent_dir/src/lib.rs"]
```

I don't think `--framework=rust` should be necessary. Does your experience suggest it is necessary?